### ISSUE TITLE
RuntimeExceptionandroid.app.ActivityThread in handleBindApplication fatalUnable to create application org.mozilla.focus.FocusApplication: java.util.MissingResourceException: Couldn't find 3-letter country code for EA Issue #

### ISSUE THREAD

**Issue by boek** [mozilla-mobile/focus-android#3604]
```
MissingResourceException

Couldn't find 3-letter country code for EA

    org.mozilla.focus.utils.ExperimentsKt in getActiveExperimentNames at line 27
    org.mozilla.focus.telemetry.TelemetryWrapper in recordActiveExperiments at line 821
    org.mozilla.focus.telemetry.TelemetryWrapper in init at line 262
    org.mozilla.focus.FocusApplication$onCreate$1 in doResume at line 72
    Called from: kotlin.coroutines.experimental.jvm.internal.CoroutineImpl in resume
    org.mozilla.focus.FocusApplication in onCreate at line 59
    Called from: android.app.Instrumentation in callApplicationOnCreate
```

**Comment by sblatz** [comment]
Seems like this has something to do with a locale that doesn't have a valid code. What's odd is I can't even find that locale on the supported [java locale list](https://www.oracle.com/technetwork/java/javase/java8locales-2095355.html) (perhaps that's the issue?)

**Comment by boek** [comment]
Fixed with mozilla-mobile/android-components#1146
